packages/editor/src/components/post-title/index.native.js

```diff
@@ -91,15 +91,24 @@ class PostTitle extends Component {
 				accessibilityLabel={
 					isEmpty( title )
 						? /* translators: accessibility text. empty post title. */
-						  __( 'Post title. Empty' )
+						  __( 'Title. Empty' )
 						: sprintf(
 								/* translators: accessibility text. %s: text content of the post title. */
-								__( 'Post title. %s' ),
+								__( 'Title. %s' ),
```

Comment:
Just wondering, what's the rationale for this change?

Reply:
Since they could be editing the "Page Title" I wanted to remove the `post_type` part. It would also make things shorter. 

Should we instead take the post_type into account and say "Page title" or "Post title" depending on the post_type that the user is editing?

Reply:
> Since they could be editing the "Page Title"

Good point!

> Should we instead take the post_type into account and say "Page title" or "Post title" depending on the post_type that the user is editing?

Maybe being specific is good, since "Title" might be confused with "Headings" by users.

Reply:
Can you explain a bit more about what your concern when it comes to Headings? 

I noticed that in the web side the placeholder says "Add title" for both post and pages.

Reply:
Opening a new post, VoiceOver focuses on the Title, and speaks out that the TextField it's focused is a Title.
That's good and works for both post and pages.

My only concern is the user expectation or knowledge about what this title is. They don't have the visual clue for it to be the post wide title or a section/paragraph "title". It might be confused with a content Heading. So "Post Title" and "Page Title" would cue better that this is the global title.

It's not a strong opinion, just something we could consider. I might be overthinking it :)
Let me know if you prefer to leave as it is to ✅ and :shipit: 👍

Reply:
My preferences is to just use "Title" it as it is. But I don't have strong feeling about this. 
But I do agree that it makes generic. Will try to update it to use "Post title" and "Page title"